code/game/atoms.dm

```diff
@@ -597,6 +597,14 @@
 
 	SEND_SIGNAL(src, COMSIG_PARENT_EXAMINE, user, .)
 
+	//MONKESTATION EDIT
+	if(user.circuit_goggles && iscarbon(user))
```

Comment:
Does this need to be all the way down in the base atom level?
That's a new if check for every single examine in the game

Reply:
Would there be a better way to handle it? I did the whole examine check based on how the research goggles did it, but on the atom level because I wanted it to target everything. 

Maybe I can do it similar to telepathy, where you click the button then click what you want to target.